Modules/_tracemalloc.c

```diff
@@ -1,4 +1,5 @@
 #include "Python.h"
+#include "pycore_tracemalloc.h"  // _PyTraceMalloc_IsTracing
```

Comment:
Some files missed imports of internal headers if they were indirectly included by Argument Clinic.